tests/resources/test-types.json

```diff
@@ -1063,7 +1088,9 @@
       },
       {
         "id": "36",
-        "linkedIds": [],
+        "linkedIds": [
+          
```

Comment:
why empty space?

Reply:
the JSON formatter just modified extra content on the file. Will remove the extra white spaces.